### ISSUE TITLE
Fireproof versions of forestry wood cannot be used to make drawers

### ISSUE THREAD

**Issue by gchpaco** [gchpaco/minecraft_eso_paco#6]
It looks like this is on Storage Drawers end, but e.g. Sequoias are fireproof by default and cannot be fashioned into drawers.

**Comment by esotericist** [comment]
Investigating this to see if there's any discussion of it upstream

**Comment by esotericist** [comment]
Not currently in game, but maybe you already checked: is there a sequoia drawer?

**Comment by gchpaco** [comment]
Yes, this came up during my abortive treehouse run, where I was living on structures around a cluster of sequoias and couldn't use their lumber to make drawers because it was fireproof.

**Comment by esotericist** [comment]
So the object actually exists, it just doesn't accept the wood?

**Comment by gchpaco** [comment]
Yup.

**Comment by esotericist** [comment]
This looks to be universally the case with fireproof wood and Extra Drawers.

I am not fixing this by hand in crafttweaker, it needs to be handled upstream.
